common/src/main/java/whocraft/tardis_refined/common/block/door/BulkHeadDoorBlock.java

```diff
@@ -94,23 +114,14 @@ public BlockEntity newBlockEntity(BlockPos blockPos, BlockState blockState) {
     public void onPlace(BlockState blockState, Level level, BlockPos blockPos, BlockState blockState2, boolean bl) {
         super.onPlace(blockState, level, blockPos, blockState2, bl);
 
-        if (blockState.getValue(OPEN)) {
-            changeBlockStates(level, blockPos, blockState, Blocks.AIR.defaultBlockState());
-        } else {
-            changeBlockStates(level, blockPos, blockState, Blocks.BARRIER.defaultBlockState());
-        }
+        changeBlockStates(level, blockPos, blockState, blockState.getValue(OPEN));
```

Comment:
Just for safety, it may be best to check if the blockstate has the open property before operation